packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -28,19 +31,24 @@ const EDiscovery = WebexPlugin.extend({
   /**
    * Creates a compliance report with a specific set of search parameters
    * @param {Object} reportRequest - A set of criteria for determining the focus of the search
+   * @param {Object} options - can specify this option :
+   *                           timeoutMs - connection timeout in milliseconds, defaults to 10s
    * @returns {Promise<ResponseEntity>} Http response containing the new report record
    */
-  createReport(reportRequest) {
+  createReport(reportRequest, options = {}) {
```

Comment:
Rather than rely on lodash, can you use `createReport(reportRequest, options = {timeoutMs: defaultTimeoutMs})` and then the spread operator in the body?

Reply:
Hi Matt, I don't understand how I can use the spread operator in this content ( due to lack of familiarity with it ). If you wouldn't mind pasting in a snippet i can update to using it, thanks

Reply:
in some of the methods options can have up to 3 properties, getContent is a good example. 

I want to allow for options to be passed in as e.g. {offset:0, size:100} and then have the timeoutMs added to this object if it is undefined in the object supplied to the method.

I also wanted to keep a consistent approach for all the methods.

Reply:
maybe something like this in the body ? : 
```
const defaultSettings = {timeoutMs: defaultTimeoutMs}
options = {...options, ...defaultSettings}
```

Reply:
updated now, have a look see what you think